packages/components/src/ComboBox/src/ComboBox.tsx

```diff
@@ -302,15 +321,16 @@ function ComboBox<T extends object>(props: ComboBoxProps<T>, ref: ForwardedRef<H
                                 ref={mergedInputRefs}
                                 className={inputClassNames}
                                 placeholder={placeholder}
+                                {...inputProps}
                             />
-                            <Button className={buttonClassNames} ref={buttonRef}>
+                            <Button className={buttonClassNames} ref={buttonRef} {...buttonProps}>
                                 <ButtonIcon size="sm" className={styles["hop-ComboBox__button-icon"]} />
                             </Button>
                         </Group>
-                        <Popover 
-                            {...otherPopoverProps} 
-                            placement={placement} 
-                            isNonDialog 
+                        <Popover
+                            {...otherPopoverProps}
```

Comment:
I added a comment in the ticket, but we're supposed to put the other props after all the props. This means that some places already done may need to be updated, sorry.

Reply:
https://gsoftdev.atlassian.net/browse/DS-360

A bit out of scope, so I created a new JIRA